### ISSUE TITLE
Thread Expander

### ISSUE THREAD

**Issue by TetraTheta** [Vencord/plugin-requests#281]
### Plugin Description

In chat channel, people can open a thread like this:

![1](https://github.com/Vencord/plugin-requests/assets/37061801/8223b310-d22a-49d0-9437-c059b988dac7)

But if the thread don't have frequent chat, it will be automatically 'folded'.

![2](https://github.com/Vencord/plugin-requests/assets/37061801/9cd3efcc-6840-4e70-a691-b9637758d0a5)

Since Discord doesn't have any option for disabling this behavior, I'd like to ask creating a plugin that prevents 'thread folding' for certain channel or thread.

Maybe this can be also be applied to forum thread which also be 'folded' when there is no activity in the thread.

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Xiarno** [comment]
I agree with this.
I tried to follow some channel to keep track of them for later uses, but many of them don't have frequent chat so they disappear and it get hard to track them back.

An option to disallow this behavior or / and pin the threads would be best!

**Comment by ENTPRESTIGIOUS** [comment]
I also agree since my threads keep disappearing after inactivity and I don't like that happening.

**Comment by TetraTheta** [comment]
https://github.com/Vencord/plugin-requests/issues/902 is closed as 'rejected (not useful / low quality / too niche)'. I wonder if this FR should be closed too or not, because this FR and the closed FR shares same 'request' that doesn't hide thread in channel list because of inactivity.

**Comment by Xiarno** [comment]
> #902 is closed as 'rejected (not useful / low quality / too niche)'. I wonder if this FR should be closed too or not, because this FR and the closed FR shares same 'request' that doesn't hide thread in channel list because of inactivity.

I don't understand why it was even closed "Not useful / Low quality / Too niche"?. For the Low quality, idk about that one. For the too niche, probably, but for the people that are in discord server that actively uses threads, this would be greatly helpful. I've lost many thread due to them disappearing, even though I subscribed to them. Not useful? See my previous point.
